### DISCUSSION TITLE
About the 3D reconstruction of non-circular plate-shaped body use FDK

### DISCUSSION THREAD

**Discussion by PeterPaang4** [astra-toolbox/astra-toolbox#508] [Q&A]
Hello, author. I am currently using the fdk algorithm to realize the three-dimensional reconstruction of a non-circular plate-shaped body, but the details of the reconstruction results are not clear. I have tried and experimented with the filtering and preprocessing of the reconstruction process, but the effect has not been achieved. I am considering whether this is because I have not performed rotation correction, but I don’t know much about Astra’s algorithm for rotation correction. Do you have any suggestions on this issue? Thank you.

**Comment by askorikov** [comment]
Hi! I am not sure what you mean by rotation correction.
